.github/dependabot.yml

```diff
@@ -5,7 +5,7 @@ updates:
     schedule:
       interval: daily
     versioning-strategy: increase
-    open-pull-requests-limit: 10
+    open-pull-requests-limit: 5
```

Comment:
i think the strategy should be to auto merge minor version increases and wait for major version increases tbh

Reply:
The auto merging is done via mergify

Reply:
ye, i think that should be the way. we already auto merge non-major version upgrades. if the problem is that we don't have capacity to address major version ugprades, then we should just disable it.